.travis.yml

```diff
@@ -122,6 +122,9 @@ before_script:
   - tar xzf tar-output/ledgersmb-travis.tar.gz
   - cd ledgersmb
   - cp conf/ledgersmb.conf.default ledgersmb.conf
+  - if [ "${DOJO_BUILT}" != "" ]; then
+      sed -r -i -e "s/^( |\t|#)*dojo_built( |\t)*=.*/dojo_built = ${DOJO_BUILT}/" ledgersmb.conf ;
+    fi
```

Comment:
Why do we need to do this?
we have 2 ledgersmb.conf files to cover dojo built and unbuilt.
Also we can specify which file is used in an env-var